buildtools/ci.template.yml

```diff
@@ -116,30 +125,13 @@ Resources:
             Version: '2012-10-17'
             Statement:
               - Action:
-                - 'logs:CreateLogGroup'
-                - 'logs:PutLogEvents'
-                - 'logs:CreateLogStream'
-                Effect: Allow
-                Resource:
-                  - !Sub "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/codebuild/${CodeBuildProjectName}"
-                  - !Sub "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/codebuild/${CodeBuildProjectName}:*"
-              - Action:
-                - 'sts:AssumeRole'
-                Effect: Allow
-                Resource:
-                  - !Ref TestRunnerRoleArn
-              - Action:
-                - codebuild:BatchPutTestCases
-                - codebuild:CreateReport
-                - codebuild:CreateReportGroup
-                - codebuild:UpdateReport
-                - codebuild:UpdateReportGroup
+                - '*'
```

Comment:
Could this be more restrictive than `*`/`*`?

https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege

Reply:
The test runner accounts were already using an IAM role that was highly permissive (`*`/`*`) which was created manually instead of using the template. This is needed because the alternative is to go into every single integration test and quantify which permissions are needed per test. Also, anytime we add new tests that could require additional permissions, we'd have to update our build infra, which is something we don't want to do. I fully acknowledge the best practice, however in our use case, it would become an ops nightmare for us to maintain the list of permissions.